src/function/algebra/simplify/util.js

```diff
@@ -14,56 +14,109 @@ export const createUtil = /* #__PURE__ */ factory(name, dependencies, ({ Functio
   // e.g. multiply is not commutative for matrices
   // The properties should be calculated from an argument to simplify, or possibly something in math.config
   // the other option is for typed() to specify a return type so that we can evaluate the type of arguments
-  const commutative = {
-    add: true,
-    multiply: true
+
+  /* So that properties of an operator fit on one line: */
+  const T = true
+  const F = false
+
+  const defaultName = 'defaultF'
+  const defaultContext = {
+    /*      */ add: { trivial: T, total: T, commutative: T, associative: T },
```

Comment:
nice trick to make these lines better readable 😄

Reply:
I like to line things up. ;-)